spec/index.html

```diff
@@ -442,42 +444,43 @@ <h3>Quoted Literals</h3>
         preceded by two concatenated <code>^</code> characters,
-        each having the code point <span class="codepoint">U+005E</span>.
-        The datatype IRI in Turtle may be written using either a
-        <a data-cite="RFC3986#section-5">resolved</a> IRI,
-        a <a data-cite="RDF12-CONCEPTS#dfn-relative-iri">relative IRI reference</a>,
-        or a <a>prefixed name</a>.
+        each having the code point <code class="codepoint">U+005E</code>.
+        A datatype IRI in Turtle may be written using either a
+        <a href="#sec-iri">resolved IRI</a>,
+        a <a href="#relative-iri">relative IRI reference</a>,
+        or a <a href="#prefixed-name">prefixed name</a>.
         If there is no datatype IRI and no language tag,
         the datatype is <code>xsd:string</code>.
       </p>
 
-      <p>A backslash ('<code>\</code>', <span class="codepoint">U+005C</span>)
+      <p>A '<code>\</code>' (backslash, code point <code class="codepoint">U+005C</code>)
         may not appear in any quoted literal except as part of an escape sequence.
         Other restrictions depend on the delimiter:</p>
 
       <ul>
-        <li>Literals delimited by <code>'</code> (<span class="codepoint">U+0027</span>)
-          may not contain unescaped <code>'</code>,
-          <code title="LINE FEED"><sub>LF</sub></code> (<span class="codepoint">U+000A</span>),
-          nor <code title="CARRIAGE RETURN"><sub>CR</sub></code> (<span class="codepoint">U+000D</span>)
+        <li>Literals delimited by <code>'</code> (apostrophe, code point <code class="codepoint">U+0027</code>)
+          may not contain unescaped
+          <code>'</code> (apostrophe, code point <code class="codepoint">U+0027</code>),
+          <code>LF</code> (line feed, code point <code class="codepoint">U+000A</code>), nor
+          <code>CR</code> (carriage return, code point <code class="codepoint">U+000D</code>)
           characters.</li>
-        <li>Literals delimited by <code>"</code>, may not contain unescaped
-          <code>"</code>,
-          <code title="LINE FEED"><sub>LF</sub></code>,
-          nor <code title="CARRIAGE RETURN"><sub>CR</sub></code>
+        <li>Literals delimited by <code>&quot;</code>, may not contain unescaped
+          <code>&quot;</code> (quotation mark, code point <code class="codepoint">U+0022</code>),
+          <code>LF</code> (line feed, code point <code class="codepoint">U+000A</code>), nor
+          <code>CR</code> (carriage return, code point <code class="codepoint">U+000D</code>)
           characters.</li>
-        <li>Literals delimited by <code>'''</code> may not contain the sequence of characters
-          <code>'''</code>.</li>
+        <li>Literals delimited by a sequence of three <code>'</code> (apostrophe, code point <code class="codepoint">U+0027</code>) may not contain a sequence of three <code>'</code> (apostrophe, code point <code class="codepoint">U+0027</code>).</li>        
```

Comment:
It looks better to actually show the syntax `'''`.

Reply:
Perhaps...?
```suggestion
        <li>Literals delimited by a sequence of three <code>'</code> (apostrophe, code point <code class="codepoint">U+0027</code>), i.e., <code>'''</code>, may not contain such a sequence.</li>        
```

Reply:
or
```suggestion
        <li>Literals delimited by a sequence of three "<code>'</code>" (apostrophe,
          code point <code class="codepoint">U+0027</code>), i.e.,
          "<code>'''</code>", may not contain such a sequence.</li>        
```

Reply:
> different colors are insufficient

Putting "..." around `'''` makes it confusing, especially if you factor in, not being colour sensitive.

Reply:
```suggestion
        <li>Literals delimited by a sequence of three <code>'</code> (apostrophe,
          code point <code class="codepoint">U+0027</code>), i.e.,
          <code>'''</code>, may not contain such a sequence.</li>    
```

Reply:
We do tend to quote single characters within the spec, although not entirely consistency right now. The EBNF also quotes these as tokens, so there is precedent. I'll update so that all the characters or sequences in this list are quoted, and generally update to favor the use of double quotes.

Reply:
I changed these to make them simpler, but still precise. I think we're getting carried away with being overly precise to the point that it makes it actually harder to understand.

> Literals delimited by "<code>'''</code>" (sequence of three apostrophes, code point <code class="codepoint">U+0027</code>) may not contain "<code>'''</code>".

Reply:
In the grammar table, STRING_LITERAL* has, for example:

> The characters between the outermost '"""'s

No colours.

Two of the boxes have open parens with no close.

Reply:
See 0cd2f05ff59526470d8a8573a28d354b913948d4.